### ISSUE TITLE
Write README + architecture overview

### ISSUE THREAD

**Issue by rogoson** [rogoson/Deep-RL-Improved#15]
Do slides and .MD?.
- Quasi MDP diagram and reward loop - Done
- PPO (structure, training loop, how works) - Done
- TD3 (structure, training loop, how works) - Done
- Feature extraction flow (LSTM), structure, loss, full headed arch. - Done
- Portfolio mathematics, rewards, CNN extraction flow, strucutre (influcenced by alexnet). - Done
- Explain noise injection + stability (motivated by paper), auc methodology - Done

Stuff missed out:
- normalisation - Done
- better hyperparam tuning - Done
